src/app/Http/Controllers/Auth/ResetPasswordController.php

```diff
@@ -58,4 +63,20 @@ public function showResetForm(Request $request, $token = null)
             ['token' => $token, 'email' => $request->email]
         );
     }
+
+    public function broker()
+    {
+        $passwords = config('backpack.base.passwords')
+            ?: config('auth.defaults.passwords');
```

Comment:
Added the newline here to keep width of the line under 80 characters. I also did it in some other places where it may not be needed. I thought it was readable like that. But they can be removed of course :)